charts/tests/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM bats/bats:1.9.0
```

Comment:
The directory structure should either be:

1:

```
charts/
  xrd-control-plane/
    templates/
      tests/
  xrd-vrouter/
    templates/
      tests/
```

Following https://helm.sh/docs/topics/chart_tests/.

Then drop the cp_ and vr_ prefixes from the files (could also drop from the test names too if you want).

or 2:

```
charts/
tests/
```

i.e., `tests` is a top-level directory.

It's important that `tests` is not directly under `charts` (each subdir in `charts` should form a Helm chart).

I have a preference for 1 - I think this is simpler for the time being, matches expectations (i.e., follows official Helm docs), and matches how these tests are structured (with separate tests for CP and VR).

Reply:
Went with option 2 due to common test utils